dependencyManagement/build.gradle.kts

```diff
@@ -2,7 +2,7 @@ plugins {
   `java-platform`
 }
 
-val otelInstrumentationVersion = "2.17.1-alpha"
+val otelInstrumentationVersion = "2.18.0-alpha"
```

Comment:
Can be removed once the renovate bot bumped the version. The new 2.18.0 version includes the https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.52.0 release that fixes the double/single quote issues.
open-telemetry/opentelemetry-java#7433